migrations/versions/0132_add_sms_prefix_setting.py

```diff
@@ -15,7 +15,7 @@
 
 def upgrade():
     op.add_column('services', sa.Column('prefix_sms', sa.Boolean(), nullable=True))
-    op.add_column('services_history', sa.Column('prefix_sms', sa.Boolean(), nullable=True))
+    op.add_column('services_history', sa.Column('prefix_sms', sa.Boolean(), nullable=True, server_default='t'))
```

Comment:
Is the server default correct for Postgres?
https://www.postgresql.org/docs/11/datatype-boolean.html

Reply:
I believe this is the correct syntax to set `server_default='t'`; in the docs it states "Unique prefixes of these strings are also accepted, for example t or n"